core/src/value/primitive.rs

```diff
@@ -923,7 +923,7 @@ impl PrimitiveValue {
     /// If the value is a string or sequence of strings,
     /// the first string is parsed to obtain an integer,
     /// potentially failing if the string does not represent a valid integer.
-    /// The string is stripped of trailing whitespace before parsing,
+    /// The string is stripped of leading/trailing whitespace before parsing,
     /// in order to account for the possible padding to even length.
```

Comment:
The idea of the subsequent line is not 100% true anymore, since the value may just have many leading and trailing spaces for whatever reasons, so I suggest we remove it. Same thing for the other occurrences.

```suggestion
    /// The string is stripped of leading/trailing whitespace before parsing.
```

Reply:
fixed it. It was only for this block or the others as well?